blocks/results-list-block/_index.scss

```
@@ -0,0 +1,15 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-result-list {
+	&--show-image {
+		@include scss.block-properties("result-list-show-image");
```

Same here - plural, and all this file - you'll also need to update the token file too

```suggestion
		@include scss.block-properties("results-list-show-image");
```